### PR TITLE
docs(Menu): create readme

### PR DESCRIPTION
**Pull request by NikitaCG**

### CODE REVIEW

File: src/components/Menu/README.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+</Menu>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
+
+| Name          | Description                             | Default |
+| :------------ | :-------------------------------------- | :-----: |
+| **size**      | Menu size `s - m - l - xl`              |    m    |
```
Comment (amje) [inline src/components/Menu/README.md:79]: This table should have 4 columns: name, description, type, default. No bold in name column, no dashes for empty defaults, types and values wrapped in code

File: src/components/Menu/README.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+| **children**  | Child element `React.ReactNode`         |    -    |
+| **className** | HTML `class` attribute `string`         |    -    |
+| **style**     | HTML `style` attribute `CSSProperties`  |    -    |
+| **qa**        | HTML `data-qa` attribute, used in tests |    -    |
+
+# Menu.Item
```
Comment (amje) [inline src/components/Menu/README.md:85]: Level 1 header must be only one on a page. Let's describe item and group in main sections

File: src/components/Menu/README.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+```tsx
+<Menu>
+  <Menu.Item disabled>First</Menu.Item>
+  <Menu.Item>Second</Menu.Item>
+</Menu>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Selected
```
Comment (amje) [inline src/components/Menu/README.md:157]: Selected and disabled can be combined into `State` section

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Menu/README.md (hunk 1 above)
Comment (NikitaCG) [reply src/components/Menu/README.md:79]: I was guided on https://github.com/gravity-ui/uikit/tree/main/src/components/Label. It probably needs refactoring.

Comment (NikitaCG) [reply src/components/Menu/README.md:79]: And I see bold in name in `Button` component README too,  where is correct ??

Comment (amje) [reply src/components/Menu/README.md]: Label needs refactoring indeed. Bolds in Button are also incorrect, everything else is ok.

Comment (NikitaCG) [reply src/components/Menu/README.md]: Ок, I'll fix it

Comment (NikitaCG) [reply src/components/Menu/README.md]: done

**Review by amje — APPROVED** [review]

**Pull request merged**